src/core/project/qgsproject.h

```diff
@@ -653,7 +653,7 @@ class CORE_EXPORT QgsProject : public QObject, public QgsExpressionContextGenera
      *
      * \since QGIS 2.4
      */
-    QgsLayerTreeGroup *createEmbeddedGroup( const QString &groupName, const QString &projectFilePath, const QStringList &invisibleLayers, QgsProject::ReadFlags flags = QgsProject::ReadFlags() );
+    QgsLayerTreeGroup *createEmbeddedGroup( const QString &groupName, const QString &projectFilePath, const QStringList &invisibleLayers, const QStringList &uncheckedGroups, QgsProject::ReadFlags flags = QgsProject::ReadFlags() );
```

Comment:
this an API break, no?

Reply:
Yes, as I had to add the additionnal argument in each call (qgisapp.cpp, qgsproject.cpp)

Reply:
We should not break the API where it can be avoided.
You should add the additional argument at the end of the signature with a default value.

Reply:
I fixed it !